core/field_angle.js

```diff
@@ -155,7 +155,8 @@ Blockly.FieldAngle.prototype.showEditor_ = function() {
   this.line_ = Blockly.utils.createSvgElement('line',
       {'x1': Blockly.FieldAngle.HALF,
       'y1': Blockly.FieldAngle.HALF,
-      'class': 'blocklyAngleLine'}, svg);
+      'class': 'blocklyAngleLine',
```

Comment:
Thanks for the review @rachel-fenichel. Just to confirm, should I change the  '.blocklyAngleLine' in the css.js file directly. Is it advisable to do so, because I was afraid it will change its behaviour globally? 
Also after making required changes, I should commit in the same branch and send a pull request, right?
P.S. This is my first pull request.